cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
is this needed?

Reply:
It was generated automatically, I didn't add it, but I believe it is need.

Reply:
I am a little worried about checking in auto-genned demo code if it is not needed, but if you think it will be useful soon-ish, i'm ok with it

Reply:
I'm ok with this code, it is documenting what is possible.